vendor/integrations.py

```diff
@@ -19,14 +21,28 @@ def get_instance(self):
             return Credential.objects.get(name=self.NAME, site=self.site)
         except ObjectDoesNotExist:
             return None
-    
+
     def save(self, data):
         if not self.instance:
             form = self.form_class(data)
         else:
             form = self.form_class(data, instance=self.instance)
-        
-        authorizenet = form.save(commit=False)
-        authorizenet.name = self.NAME
-        authorizenet.site = self.site
-        authorizenet.save()
\ No newline at end of file
+
+        integration_form = form.save(commit=False)
+        integration_form.name = self.NAME
+        integration_form.site = self.site
+        integration_form.save()
+
+
+class AuthorizeNetIntegration(BaseIntegration):
+    NAME = "AuthorizeNet Integration"
+
+    site = None
+    form_class = AuthorizeNetIntegrationForm
+
+
+class StripeIntegration(BaseIntegration):
+    NAME = "Stripe Integration"
+
+    site = None
+    form_class = StripeIntegrationForm
```

Comment:
I don't know if there are already a unit test for the AuthorizeNetIntegration, but can you add one to the StripeIntegration class

Reply:
On it